prow/deck/deployment.yaml

```diff
@@ -23,7 +23,7 @@ spec:
       terminationGracePeriodSeconds: 30
       containers:
         - name: deck
-          image: gcr.io/k8s-prow/deck:v20190307-bbad344
```

Comment:
Have there been any changes to the `images` folder to warrant this change? (I suspect the images should be identical bar the version tag)

Reply:
@JoelSpeed See my comment on hook's manifest - it's the same idea, the image's server has been improved with extra endpoints for health and ready checks. For some reason, I cannot find the upstream PR that added them anymore. Will edit this comment if I find it again.

Reply:
@JoelSpeed well I promised to include the PR once I've found it again so here :)

https://github.com/kubernetes/test-infra/pull/11659